compiler/rustc_privacy/messages.ftl

```diff
@@ -8,6 +8,7 @@ privacy_from_private_dep_in_public_interface =
 privacy_in_public_interface = {$vis_descr} {$kind} `{$descr}` in public interface
     .label = can't leak {$vis_descr} {$kind}
     .visibility_label = `{$descr}` declared as {$vis_descr}
+    .suggestion = consider adding `pub` in front of it
```

Comment:
this wording is a bit awkward, perhaps "consider making the`{$descr}` public" or something.

i also don't know if this will unilaterally be applicable. have you tested combinations of items that are already `pub(crate)` but being exported in `pub` signatures, for example?

Reply:
I agree that it does sound awkward. So I went ahead with your suggestion.

I am not sure what you mean by that example. Can you show that as code?

Reply:
I agree with you so I changed the message to what you suggested.

Also, I am not sure what you mean by that example, sorry